vaadin-grid-pro-flow-integration-tests/src/test/java/com/vaadin/flow/component/gridpro/test/BasicIT.java

```diff
@@ -26,6 +26,12 @@ public void editColumnsAdded() {
         Assert.assertEquals(columns.size(), 3);
     }
 
+    @Test
+    public void customRepresentationIsRendered() {
+        GridTHTDElement cell = grid.getCell(0, 2);
+        Assert.assertTrue(cell.innerHTMLContains("<span>No</span>"));
+    }
+
```

Comment:
Add another test which changes the value and verifies the representation after that.